components/org.wso2.carbon.identity.oauth/src/test/resources/dbScripts/identity.sql

```diff
@@ -1298,6 +1298,15 @@ CREATE TABLE IF NOT EXISTS AUTHORIZED_SCOPE(
     CONSTRAINT AUTHORIZED_SCOPE_UNIQUE UNIQUE (APP_ID, SCOPE_ID)
     );
 
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_TOKEN_CLAIMS (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    CONSUMER_KEY VARCHAR(255) NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    CLAIM_URI VARCHAR(255),
+    CONSTRAINT TOKEN_CLAIMS_CONSTRAINT UNIQUE (CONSUMER_KEY, CLAIM_URI),
```

Comment:
Consumer Key can be reused accorss tenants right?, can we add tenant ID into this consrtaint

Reply:
fixed